### PR TITLE
RIA-2810: Removes the TBC placeholder in deadlines set by TCW's.

### PR DESCRIPTION
**Pull request by alexbaeza**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RIA-2810

### Change description ###
**RIA-2810:** 
- Removes the TBC place holder in deadlines set by TCW's.
- Loads directions into the session.
- This is needed in order to display deadlines set by tcw's on directions.

Requires CCD Definition:
https://github.com/hmcts/ia-ccd-definitions/pull/100

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: app/utils/event-deadline-date-finder.ts [hunk 1]
```diff
@@ -4,26 +4,57 @@ import moment from 'moment';
...
+ * Finds a targeted direction, retrieves it's due date and returns it as a string with the correct date format
+ * @param directions the directions
+ * @param directionTagToLookFor the direction to find
+ */
+function getFormattedDirectionDueDate(directions: Direction[], directionTagToLookFor: string) {
+  let formattedDeadline = null;
+  const direction = directions.find(d => d.tag === directionTagToLookFor);
+  if (direction) {
+    const dueDate = direction.dueDate;
+    formattedDeadline = moment(dueDate).format('DD MMMM YYYY');
```
Comment (chrisgrimble) [inline app/utils/event-deadline-date-finder.ts]: Can we pull this out to a util class think we are going to be showing dates like this all over the place soon.

File: app/utils/event-deadline-date-finder.ts [hunk 2]
```diff
@@ -4,26 +4,57 @@ import moment from 'moment';
...
+      break;
     }
-    case 'awaitingRespondentEvidence':
-    case 'appealSubmitted': {
+    case 'appealSubmitted':
+    case 'awaitingRespondentEvidence': {
       const triggeringDate = history['appealSubmitted'].date;
-      const formattedDeadline = moment(triggeringDate).add(daysToWaitAfterSubmission, 'days').format('DD MMMM YYYY');
-      return formattedDeadline || null;
+      formattedDeadline = moment(triggeringDate).add(daysToWaitAfterSubmission, 'days').format('DD MMMM YYYY');
```
Comment (chrisgrimble) [inline app/utils/event-deadline-date-finder.ts]: Same again

Comment (alexbaeza) [reply app/utils/event-deadline-date-finder.ts]: Thanks,  for the suggestion I have moved the format only into date-formats.ts.

**Comment by alexbaeza** [comment]
Closing this PR due to an issue in jenkins and re-openned here:
https://github.com/hmcts/ia-aip-frontend/pull/310

**Pull request closed without merging**